yb-voyager/cmd/exportData.go

```diff
@@ -293,6 +293,9 @@ func exportPGSnapshotWithPGdump(ctx context.Context, cancel context.CancelFunc,
 	if err != nil {
 		return fmt.Errorf("export snapshot: failed to create replication slot: %v", err)
 	}
+	utils.PrintAndLog(color.YellowString(fmt.Sprintf("Created replication slot '%s' on source PG database. "+
```

Comment:
Can we make it bold and blink?
```
// Base attributes
const (
	Reset Attribute = iota
	Bold       <----------------
	Faint
	Italic
	Underline
	BlinkSlow   <-----------
	BlinkRapid
	ReverseVideo
	Concealed
	CrossedOut
)
```

Reply:
Added bold. 
For some reason, blinkSlow/blinkRapid wasn't working.